templates/index.html

```diff
@@ -132,8 +132,12 @@ <h2 class="subtitle has-text-centered">その他設定</h2>
               <div class="field-body">
                 <div class="field">
                   <div class="control is-small">
-                    <label class="checkbox">
-                      <input type="checkbox" id="sound" name="sound">
+                    <label class="select">
```

Comment:
ほかにあわせたいですー

<img width="497" alt="スクリーンショット 2021-04-30 23 13 58" src="https://user-images.githubusercontent.com/8909592/116707541-c31fc980-aa09-11eb-92f5-90923ced15ca.png">


```suggestion
                    <label class="select is-small">
```